packages/calcite-components/src/components/input-time-zone/utils.ts

```diff
@@ -4,6 +4,48 @@ import { InputTimeZoneMessages } from "./assets/input-time-zone/t9n";
 
 const hourToMinutes = 60;
 
+const timeZoneNameBlockList = [
```

Comment:
Is there a way to get this list from a trusted source rather than having to maintain it ourselves?

Reply:
Good idea. I could reliably exclude `Etc/X` ones, but would have to investigate separately if entries with no delimiter are reliable to block as well.

Reply:
I focused too much on having this dynamically generated that I didn't address the "trusted source" part of your question. Unfortunately, `Intl` doesn't provide metadata for time zone names, so we have to handle this on our side. That would definitely make things simpler here.